payments-core/build.gradle

```diff
@@ -16,6 +16,7 @@ configurations {
 }
 
 dependencies {
+    implementation project(":paymentsheet")
```

Comment:
@michelleb-stripe 
FYI with this added to `payments-core`, `paymentsheet` module should have a dedicated `aar` published to maven for each release.

It's done by applying `apply from: "${rootDir}/deploy/deploy.gradle"` in the paymentsheet module's `build.gradle` and define the following three variables for the target name, please see [here](https://github.com/stripe/stripe-android/blob/96d4ac4ae20b8b59d202a3bfa99b52084363b1a2/payments-core/build.gradle#L153) for example on payments-core
```
ext {
    artifactId = "payments-core"
    artifactName = "payments-core"
    artifactDescrption = "The core module of Stripe Payment Android SDK"
}
```
We would  also need to include `paymentsheet` module for dokka, ktlint, ci unittests now that it's officially released.

These changes could be done in a separate PR, happy to help out when this is submitted :)

Reply:
Is there a way to just bundle the content of `paymentsheet` into `payments-core`? We want this dependency to be inverted, but need some refactors for that, so this is how we can make it work now.

Reply:
hmm don't think it's possible unless you use some fat aar plugin, by inverted do you mean all paymentsheet related code moved into `paymentsheet` module?

Reply:
another possible solution is to submit the change as is but invert them before the next release

Reply:
Yes, I was thinking about that. It's not too hard to comment out the places where we depend on classes from `paymentsheet`. Only `BaseAddPaymentMethodFragment` and `SupportedPaymentMethod` I think.

Reply:
If that's the case, I would suggest moving the code from `payments-core` to `paymentsheet` firs, since that sounds like the work we would need to to anyway before next release